ibis/expr/api.py

```diff
@@ -4116,7 +4116,8 @@ def mutate(table, exprs=None, **mutations):
     )
 
     by_name = collections.OrderedDict(
-        (expr.get_name(), expr) for expr in exprs
+        (expr.get_name(), expr) if expr.has_name() else (None, expr)
```

Comment:
does get_name return None ? can you just change it to?

Reply:
Good point. Updated!